### PR TITLE
Bug 2043572: Deleting namespace store in UI secrets remains

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Random characters added to the secret at the end

Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh , @GowthamShanmugam , @SanjalKatiyar Please review

File: packages/shared/modals/DeleteModal.tsx [hunk 1]
```diff
@@ -55,6 +61,29 @@ const DeleteModal: React.FC<CommonModalProps<DeleteModalExtraProps>> = ({
       });
   });
 
+  const getSecName = (resource: K8sResourceCommon) => {
+    const getType = (resource) => {
+      switch (resource['spec']['type']) {
```
Comment (GowthamShanmugam) [inline packages/shared/modals/DeleteModal.tsx:66]: i deleting namespace store should deleted secret using owner ref.

Comment (TimothyAsirJeyasing) [reply packages/shared/modals/DeleteModal.tsx]: Secret named with unique

**Review by bipuladh — CHANGES_REQUESTED** [review]
Lets do this only if a secret of that particular name exists. We can handle this if we get an error around secret not being present. So the flow would be. 
1) Try and create a secret without the random characters.
2) If the secret is present then we would get an error. If we get such an error then only append these random characters to the name. We could avoid that by adding more intelligence. We could try checking the content of these two secrets and if it matches we could use the same secret instead of creating but this could be later tackled (the matching of secret content).

**Pull request closed without merging**